tests/testthat/test-merge_sce_list.R

```diff
@@ -0,0 +1,251 @@
+# helper function to add more info to a simulated SCE ----
+add_sce_data <- function(sce) {
+
+  # add some coldata columns
+  colData(sce)[["sum"]] <- runif(nrow(colData(sce)))
+  colData(sce)[["detected"]] <- runif(nrow(colData(sce)))
+  colData(sce)[["total"]] <- runif(nrow(colData(sce)))
+
+  # add some rowdata columns
+  rowData(sce)[["gene_names"]] <- rownames(rowData(sce))
+  rowData(sce)[["other_column"]] <- runif(nrow(rowData(sce)))
```

Comment:
Minor, but removing some redundancy
```suggestion
  # add some coldata columns
  colData(sce)[["sum"]] <- runif(ncol(sce))
  colData(sce)[["detected"]] <- runif(ncol(sce))
  colData(sce)[["total"]] <- runif(ncol(sce))

  # add some rowdata columns
  rowData(sce)[["gene_names"]] <- rownames(sce)
  rowData(sce)[["other_column"]] <- runif(nrow(sce))
```